src/scripts/i18n/list_source_files.py

```
@@ -0,0 +1,113 @@
+        folder = plural_map[types_requested[0]]
+        out_dir = Path(f"src/scripts/i18n/{folder}")
+    else:
+        # Multiple categories – join their plural forms to keep uniqueness
+        joined = "-".join(sorted(plural_map[t] for t in types_requested))
+        out_dir = Path(f"src/scripts/i18n/{joined}")
+    out_dir.mkdir(parents=True, exist_ok=True)
+    out_path = out_dir / "input.json"
+    with open(out_path, "w", encoding="utf-8") as out_f:
+        json.dump(files_map, out_f, indent=2, ensure_ascii=False)
+
+    print(f"Wrote {len(files_map)} files → {out_path}") 
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)